Dockerfile

```diff
@@ -151,7 +151,7 @@ COPY --from=composer /app /var/www/cms
 COPY --from=webpack /app/web/dist /var/www/cms/web/dist
```

Comment:
These files (and others) potentially have incorrect ownership now?

Reply:
No, the file permission are looking fine inside the dist folder